### ISSUE TITLE
Release v6.5.1 template fails to create a new project with npx react-native init

### ISSUE THREAD

**Issue by azizuysal** [react-native-community/react-native-template-typescript#144]
# Bug report

## Summary

npx react-native init MyApp --template react-native-template-typescript fails with:

```bash
✔ Downloading template
✔ Copying template
✔ Processing template
✖ Installing dependencies
error Error: Command failed: npm install
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @types/react-test-renderer@16.13.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'MyApp'
npm ERR! notarget 
```

## Environment info

`react-native info` output:

```bash
# paste it here
❯ npx react-native info                                                 
error Unrecognized command "info".
info Run "react-native --help" to see a list of all available commands.
```

I am using:
- macOS 10.15.5
- zsh 5.7.1
- node v12.16.0
- npm 6.14.4
- xcode 11.5

Library version: x.x.x

## Steps to reproduce

1. Open a terminal window
2. Type ```npx react-native init MyApp --template react-native-template-typescript```

Describe what you expected to happen:

1. A new react-native project created with react-native v0.63 using typescript

**Comment by darkgl0w** [comment]
Hi,
the problem comes from this line:
https://github.com/react-native-community/react-native-template-typescript/blob/f47226daa56de8509774cce41b5a3f8671f5fd38/template/package.json#L22

this package version does not exists :d

**Comment by emin93** [comment]
Should be fixed in [6.5.2](https://github.com/react-native-community/react-native-template-typescript/releases/tag/6.5.2). Can you please verify if it works now?

**Comment by azizuysal** [comment]
6.5.2 works, thank you.
